### PR TITLE
Add sat vap

### PR DESCRIPTION
**Pull request by claresinger**
addressing issue #758

### CODE REVIEW

File: PySDM/physics/constants_defaults.py [hunk 1]
```diff
@@ -63,6 +63,22 @@
...
+L77W_A1 = 4.436518521e-1 * si.hPa / si.K
+L77W_A2 = 1.428945805e-2 * si.hPa / si.K**2
+L77W_A3 = 2.650648471e-4 * si.hPa / si.K**3
+L77W_A4 = 3.031240396e-6 * si.hPa / si.K**4
+L77W_A5 = 2.034080948e-8 * si.hPa / si.K**5
+L77W_A6 = 6.136820929e-11 * si.hPa / si.K**6
+
+L77I_A0 = 6.109177956 * si.hPa
+L77I_A1 = 5.03469897e-1 * si.hPa / si.K
+L77I_A2 = 1.886013408-2 * si.hPa / si.K**2
```
Comment (claresinger) [inline PySDM/physics/constants_defaults.py:76]: The tests are failing because of the units here? I don't really understand the error message, @slayoo.

Comment (slayoo) [reply PySDM/physics/constants_defaults.py]: see https://github.com/atmos-cloud-sim-uj/PySDM/pull/759/commits/21ba41d69e687cffa327aa2814152dda9ecf5912 :)

Comment (claresinger) [reply PySDM/physics/constants_defaults.py]: Ah! Good eye!

**Comment by claresinger** [comment]
Test plot. Looks good.
e_s

**Pull request merged**